tests/VMelnalksnis.PaperlessDotNet.Tests.Integration/Documents/DocumentClientTests.cs

```diff
@@ -101,6 +104,62 @@ public async Task Create()
 		}
 	}
 
+	[Test]
+	public async Task Download()
+	{
+		if (PaperlessVersion < new Version(2, 0))
```

Comment:
If I remember correctly, paperless only returns the created document id since version 2. Before that it simply returns that the import process has started, and you have to separately poll the task status.

I'd say it's ok to skip this test for older versions, or maybe just waiting for some time after creating the document.